### ISSUE TITLE
jCenter going away - New snapshot location?

### ISSUE THREAD

**Issue by tlf30** [i-net-software/SetupBuilder#96]
With jcenter going away, will snapshots be moved to a different repository?

**Comment by Horcrux7** [comment]
We will try to move to sonatype. Currently we are in the evaluation.

**Comment by Horcrux7** [comment]
I have oversee that this project is not hosted in the JCenter maven repository. We have some other project that on JCenter. This project is hosted in the Gradle plugin repository: 'https://plugins.gradle.org/m2/'

> With jcenter going away, will snapshots be moved to a different repository?

What do you means with it?

**Comment by tlf30** [comment]
For `com.github.i-net-software:SetupBuilder:master-SNAPSHOT` I was unable to resolve the package with `jcenter()` disabled. After tracking it down it looks like this actually comes from jitpack. After deleting the .m2 cached packages I was able to successfully resolve from jitpack. This is the third jitpack package I have found that removing jcenter has broke the maven cache for some reason... 

Looks like it is just an issue on my end, I think this can be closed.
